lib/Pod/Simple/XHTML.pm

```diff
@@ -131,11 +131,12 @@ default.
 
 In turning C<< L<crontab(5)> >> into http://whatever/man/1/crontab, what
 to put before the "1/crontab". The default value is
-"http://man.he.net/man".
+"https://man7.org/linux/man-pages/man".
 
 =head2 man_url_postfix
 
-What to put after "1/crontab" in the URL. This option is not set by default.
+What to put after "1/crontab" in the URL.
```

Comment:
Should be:
```pod
What to put after "1/crontab.1" in the URL.
```
because `resolve_man_page_link` now always appends a dot and another section number to the page name.

(This is technically an incompatible change to the public interface. Even if someone wanted to keep linking to man.he.net, they couldn't do it by just setting `man_url_prefix` and `man_url_postfix`.)